### PR TITLE
add doc for guice stage

### PR DESCRIPTION
**Pull request by guillaumecle**

### CODE REVIEW

File: src/main/resources/testng-1.0.dtd [hunk 1]
```diff
@@ -62,7 +65,7 @@ Cedric Beust & Alexandru Popescu
     verbose CDATA #IMPLIED
     parallel (false | methods | tests | classes | instances) "false"
     parent-module CDATA #IMPLIED
-    guice-stage CDATA #IMPLIED
+    guice-stage (DEVELOPMENT | PRODUCTION | TOOL) #IMPLIED
```
Comment (guillaumecle) [inline src/main/resources/testng-1.0.dtd]: Should we say that guice-stage is ```#IMPLIED``` or that its default value is ```DEVELOPMENT```?

Comment (cbeust) [inline src/main/resources/testng-1.0.dtd]: Can we specify a default in the DTD or does it have to be a comment? Either way, yes, specifying the default is the correct way to do this.

Comment (guillaumecle) [inline src/main/resources/testng-1.0.dtd]: I think this actual DTD and not a comment.

**Comment by cbeust** [comment]
Thanks!

**Pull request merged**